pubspec.yaml

```diff
@@ -14,7 +14,6 @@ dependencies:
   compiler_unsupported: "^1.22.0"
   crypto: ">=1.0.0 <3.0.0"
   glob: "^1.0.0"
-  html: "^0.13.0"
```

Comment:
I added it in an earlier commit, but was able to remove the dependency (no special html handling at all any more - it works out of the box with all the regular dart2js workflows)